package Protocol;
message C2SQQLoginReq
{
	required string account = 1;
	required string password = 2;
}